README.md

```diff
@@ -20,3 +23,5 @@ randomly without memory instead of systematically trying them?
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+The average-case runtime for my implementation would be T(n) ∈ θ(n * n!). Checking if a permutation is O(n) and that is mutiplied with a multiple of n!. A best-case input for the implementation is a sorted array. It checks if the array is sorted when entered which takes θ(n) time and if it is sorted, it simply returns 1. The worst-case is being given a reverse-sorted array. It will perform every single possible permutation of the list before being found everytime it is given a reverse sorted array, or T(n) ∈ θ(n * n!). If it were to be attempted without any memory and doing random permutations until finding the sorted one, it would be about T(n) ∈ θ(n * n!) because unless there are duplicate entries, there only exists one sorted permutation out of n! total permutations. This means the odds of finding the sorted permutation each time is 1/n! so it should require n! permutations on average when creating them randomly and takes O(n) to check if its sorted.
```

Comment:
When you're creating permutations randomly, could you not have the same ones multiple times? Is there any guarantee that the right permutation will ever be found?